bin/wabur

```diff
@@ -12,26 +12,25 @@ require 'logger'
 require 'wab'
 require 'wab/impl'
 
-cfg = {
-  'handler.path' => '/v1',
-  'type_key' => 'kind',
-  'http.dir' => '.',
-  'controller' => 'BasicController',
-  'http.port' => 6363,
-  'verbose' => Logger::WARN
+# Initial options
+options = {
+  'source'      => '.',
+  'config_file' => nil
 }
-cfg_file = nil
 verbose_cnt = 0
 
 opts = OptionParser.new(%{Usage: wabur [options]
 
-A WAB Runner that is configured with a unix style conf file.
+A WAB Runner that is configured with a UNIX style config file.
 })
-opts.on('-v', 'verbose')                                     { verbose_cnt += 1 }
-opts.on('-c', '--conf String', String, 'configuration file') { |c| cfg_file = c }
+opts.on('-s', '--source PATH', String, 'Source directory')   { |d| options['source'] = d }
```

Comment:
Sticking with the Ruby and irb conventions `-I` should be used to add to the load path. That should be allowed multiple times as well.

Reply:
I'm not improving the executable entirely in this PR..
Regarding the switch, I opted for `--source` to denote the directory path from where the app will be run.. In the `script/start-sample`, this is handled by the shell variable `$sampledir`

So what's your final suggestion..? I'm getting a sort of mixed vibe here..

Reply:
I guess it got a bit confusing. For now just a rename would be great. 'source' is easily confused with 'source code' which is not what is meant here.